.env

```diff
@@ -35,7 +35,6 @@ NEXT_PUBLIC_CONTACT_PROVIDER='gorgias'
 NEXT_PUBLIC_RECAPTCHA_SITE_KEY='6LdArl4gAAAAADf30J5nPO2hStNG8_IYrnl43RlT'
 
 # Sentry
-NEXT_PUBLIC_SENTRY_DSN='https://b3dadbce4879487b90076bf1350f8534@o4504254488313856.ingest.sentry.io/4504254630461440'
 SENTRY_ORG='penny-bz'
```

Comment:
`NEXT_PUBLIC_SENTRY_DSN` should have been in here, since it makes it impossible to override Sentry for, e.g., the preview environment.